haiku-apps/optipngtranslator/optipngtranslator-0.1.0.recipe

```diff
@@ -6,12 +6,12 @@ be available in any native application. You find settings like the \
 optimization level in the DataTranslations preferences. The higher the \
 level, the longer it takes."
 HOMEPAGE="https://github.com/noryb009/OptiPNGTranslator"
-COPYRIGHT="2013-2018 Luke (noryb009)"
+COPYRIGHT="2013-2022 Luke (noryb009)"
 LICENSE="MIT"
-REVISION="5"
```

Comment:
when creating a new version (renaming the file) you can reset REVISION to 1 instead of incrementing it